hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaDistributionSet.java

```diff
@@ -347,14 +347,14 @@ public void fireUpdateEvent(final DescriptorEvent descriptorEvent) {
                 new DistributionSetUpdateEvent(this, EventPublisherHolder.getInstance().getApplicationId()));
 
         if (isSoftDeleted(descriptorEvent)) {
-            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(),
+            publishEventWithEventPublisher(new DistributionSetDeletedEvent(getTenant(), getId(), getClass().getName(),
```

Comment:
So the fixed type is the same like the runtime type.  It's different, if we would like to use the interface, but we don't want :).